src/list.cpp

```diff
@@ -152,7 +152,7 @@ void list_probes(const BPFtrace &bpftrace, const std::string &search_input)
     if (bpftrace.pid_ > 0)
     {
       executable = get_pid_exe(bpftrace.pid_);
-      absolute_exe = executable;
+      absolute_exe = path_for_pid_mountns(bpftrace.pid_, executable);
```

Comment:
this should maybe be changed to resolve_binary_path, wdyt?

Reply:
i think i'll leave it as is for now, as this should always be safe to take the path relative to the pid's mount namespace, whether the mount namespace is the same as for bpftrace or note.

Reply:
yeah nvm, resolve_binary_path returns a vector.